src/AWSS3.jl

```diff
@@ -603,8 +604,30 @@ function s3_list_objects(
         end
     end
 end
+s3_list_objects(a...; kw...) = s3_list_objects(global_aws_config(), a...; kw...)
 
-s3_list_objects(a...) = s3_list_objects(global_aws_config(), a...)
+"""
+    s3_directory_stat([::AbstractAWSConfig], bucket, path)
+
+Determine the properties of an S3 "directory", size and time of last modification, that cannot be determined
+directly with the standard AWS API.  This returns a tuple `(s, tmlast)` where `s` is the size in bytes, and
+`tmlast` is the time of the latest modification to a file within that directory.
+"""
+function s3_directory_stat(aws::AbstractAWSConfig, bucket, path)
```

Comment:
Can we type these params?

Reply:
That is generally my preference, but I was following the conventions already existing in this package.

Reply:
I added them for this anyway.  May be worth going back one day and cleaning this up in the rest of the package.